azure-monitor-opentelemetry/README.md

```diff
@@ -95,6 +99,8 @@ Samples are available [here][samples] to demonstrate how to utilize the above co
 * [OpenTelemetry Python Official Docs][ot_python_docs]
 
 <!-- LINKS -->
+[azure_core_tracing_opentelemetry_plugin]:https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/core/azure-core-tracing-opentelemetry
+[azure_core_tracing_opentelemetry_plugin_samples]: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/core/azure-core-tracing-opentelemetry/samples
```

Comment:
Shouldn't we be using our own samples with our distro instead of using the samples of azure-core?

Reply:
That makes sense. Their samples would actually be confusing since our Cxs don't need to set the span. Will change

Reply:
Changed to point to our sample only